app/services/variant_units/option_value_namer.rb

```diff
@@ -64,36 +64,42 @@ def option_value_value_unit_scaled
     def scale_for_unit_value
       units = {
         'weight' => {
-          1.0 =>         { 'name' => 'g',  'system' => 'metric' },
-          28.35 =>       { 'name' => 'oz', 'system' => 'imperial' },
-          453.6 =>       { 'name' => 'lb', 'system' => 'imperial' },
-          1000.0 =>      { 'name' => 'kg', 'system' => 'metric' },
-          1_000_000.0 => { 'name' => 'T',  'system' => 'metric' }
+          1.0 => { 'name' => 'g', 'system' => 'metric' },
+          28.35 => { 'name' => 'oz', 'system' => 'imperial' },
+          453.6 => { 'name' => 'lb', 'system' => 'imperial' },
+          1000.0 => { 'name' => 'kg', 'system' => 'metric' },
+          1_000_000.0 => { 'name' => 'T', 'system' => 'metric' }
```

Comment:
Ah, okay, yeah - I was confused as to what rubocop wanted here, so I aligned the keys.... imo it's easier to read when the keys are lined up, rather than with the whitespace removed, but I can see both sides, so 👍

Reply:
ah, yes, looks better aligned. I just wanted to fix rubocop, we can align again and add a rubocop exception for example.